static/js/services/vulnerability-service.js

```diff
@@ -20,7 +20,7 @@ angular.module('quay').factory('VulnerabilityService', ['Config', 'ApiService',
     return null;
   }
 
-  vulnService.buildVulnerabilitiesInfo = function(image, resp) {
+  vulnService.buildVulnerabilitiesInfo = function(manifest, resp) {
```

Comment:
Change all these methods to accept the whole `Manifest` object instead of just the `Manifest.image` property.

Reply:
Awesome